### DISCUSSION TITLE
how to get item icon to use in UI?

### DISCUSSION THREAD

**Discussion by barashF** [SmartlyDressedGames/Unturned-3.x-Community#3981] [Q&A]
Good day, I'm trying to implement the display of an item's icon by id, but I don't understand where you can get it from, I would be very grateful for your help.

**Comment by GazziFX** [comment]
Open game -> Workshop -> Press F1 -> Generate All
Go to Unturned folder -> Extras -> Icons
Upload to web server
Use EffectManager.sendUIEffectImageURL

**Reply by barashF** [reply]
that is, you implement this by uploading the necessary images to the server in the directory where the plugin is located, and getting the images by entering the path to the file? I just thought you can somehow get the icon from the available methods of the game code

**Comment by iBowie** [comment]
I tried to streamline this process by creating this repo, feel free to contribute if you want too
[https://github.com/iBowie/OpenUnturnedIcons](https://github.com/iBowie/OpenUnturnedIcons)

**Reply by barashF** [reply]
that is, you implement this by uploading the necessary images to the server in the directory where the plugin is located, and getting the images by entering the path to the file? I just thought you can somehow get the icon from the available methods of the game code

**Reply by GazziFX** [reply]
Unturned doesn't have this method
https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/3010
